app/src/config.js

```diff
@@ -1,8 +1,8 @@
-const dev = process.env.NODE_ENV === `develop`
+const dev = process.env.NODE_ENV === `development`
 const stargate =
   process.env.STARGATE ||
   (dev
-    ? `localhost:8080`
+    ? `http://localhost:8080`
```

Comment:
`http` ??

Reply:
Basically by omitting it chrome tries to keep the same level of security as your page, so `https`

But our caddy file comes with no certificates, so if you have a running client and click on those links you should receive:
http://localhost:8080/node_version -> 0.31.2-20-g1aa6c197
https://localhost:8080/node_version -> ERR_SSL_PROTOCOL_ERROR